homeworks/homework-vue/src/components/NewsList/NewsList.spec.ts

```diff
@@ -0,0 +1,110 @@
+import { createLocalVue, mount } from "@vue/test-utils";
+import NewsList from "@/components/NewsList/NewsList.vue";
+import NewsItem from "@/components/NewsItem/NewsItem.vue";
+import { NewsItemModel } from "@/models/news-item.model";
+
+const newsItemsMock = [
+  new NewsItemModel({
+    id: 1,
+    title: "Hackernews Nr 1",
+    votes: 1
+  }),
+  new NewsItemModel({
+    id: 2,
+    title: "Hackernews Nr 2",
+    votes: 2
+  }),
+  new NewsItemModel({
+    id: 3,
+    title: "Hackernews Nr 3",
+    votes: 3
+  })
+];
+
+describe("NewsList", () => {
+  const localVue = createLocalVue();
+
+  it("check App-instance-existance and newsList-instance-existance", () => {
```

Comment:
Be affirmative in your test case descriptions.
```suggestion
  it("exists", () => {
```